mypy.ini

```diff
@@ -1,4 +1,14 @@
 [mypy]
```

Comment:
make sure to remove all the configuration in pre-commit and point it to this config file if necessary

Reply:
we might have to change the pre-commit hook to run the local mypy instead of installing in a new virtualenv